packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -27,18 +27,18 @@ export class RetriableDocumentStorageService extends DocumentStorageServiceProxy
         return this.readWithRetry(async () => this.internalStorageService.getSnapshotTree(version));
     }
 
-    public async read(blobId: string): Promise<string> {
-        return this.readWithRetry(async () => this.internalStorageService.read(blobId));
+    /**
+     *
+     * @deprecated - only here for back compat, will be removed after release
+     */
+    public async read(blobId: string): Promise<ArrayBufferLike> {
```

Comment:
Please either delete it or make it return base64-encoded string. There is no need to have duplicate functions and reuse name that is used for different purpose on driver interface. Same feedback as in other place - that should be compiler error, please investigate why it's not (assuming that storage interface continues to have same read() signature as it used before)